examples/SampleApp/src/hooks/useChatClient.ts

```diff
@@ -41,14 +40,22 @@ export const useChatClient = () => {
     >(config.apiKey, {
       timeout: 6000,
     });
-    const randomSeed = getRandomInt(1, 50);
-    const user = {
-      id: config.userId,
-      image: config.userImage,
-      name: config.userName,
-    };
-
-    await client.connectUser(user, config.userToken);
+
+    await streamCacheSetup(client, {
+      getItem: (key: string) => AsyncStore.getItem(key, null),
+      removeItem: (key: string) => AsyncStore.removeItem(key),
+      setItem: (key: string, value: any) => AsyncStore.setItem(key, value),
```

Comment:
still need to type it properly after the client is released and i can get the right types from there